target_chains/solana/sdk/js/pyth_solana_receiver/src/PythSolanaReceiver.ts

```diff
@@ -168,15 +181,15 @@ export class PythTransactionBuilder extends TransactionBuilder {
 
   /**
    * Add instructions to update price feed accounts to the builder.
+   * Price feed accounts are a special type of price update accounts. Instead of being ephemeral accounts, they are fixed accounts for each feed id that can only be updated with a more recent price update than the one they contain.
+   * Use this function to update a shared price feed account with a recent price
```

Comment:
Not a 100% satisfied with this explanation

Reply:
yeah... I think one way to fix this is to create a richer mental model for the reader to understand how things work. I think the model is something like:

* Pyth allows you to post any price update on-chain. This info is stored in price update accounts
* Pyth additionally supports price feed accounts, which hold a stream of price updates that move forward in time. Programs can read a price feed account to get a recent price for a specific feed id. Price feeds are built on top of pull updates (i.e., using price update accounts).

Key thing to clarify is the relationship between price feeds and price updates. Is a feed composed of price updates? is it a sequence of updatse?